lib/isEligibleRequest.js

```diff
@@ -29,7 +29,7 @@ const hasAcceptableMethod = (req) => !UNACCEPTABLE_METHODS.has(req.method);
  */
 const hasAcceptableContentType = (req) => {
   const contType = req.headers['content-type'];
-  return contType.includes('boundary=') && ACCEPTABLE_CONTENT_TYPE.test(contType);
+  return contType && contType.includes('boundary=') && ACCEPTABLE_CONTENT_TYPE.test(contType);
```

Comment:
```suggestion
  return !!contType?.includes('boundary=') && ACCEPTABLE_CONTENT_TYPE.test(contType);
```
reason for this suggestion is that, the initial implementation you provided has potential to cause the function to return the `undefined` or `null` or whatever falsy value  that `contType` could potentially be. Kindly refer to the small demo i showed in the screenshot below for reference.

![Screenshot 2023-11-21 at 2 06 04 PM](https://github.com/richardgirges/express-fileupload/assets/36765257/969889f7-8da3-49ef-b920-ca4cfa22f11b)